resources/module.rb

```diff
@@ -23,7 +27,27 @@
 
 action :enable do
   # Call the  apache2_mod_resource if we want it configured
-  send("apache2_mod_#{new_resource.name}", '') if new_resource.conf
+  if new_resource.conf
+    mod_resource_name = "apache2_mod_#{new_resource.name}"
+
+    send(mod_resource_name, 'default')
+
+    # pass Hash new_resource.mod_conf as properties
+    begin
+      r = resources(mod_resource_name.to_sym => 'default')
+
+      new_resource.mod_conf.each do |k, v|
```

Comment:
I would prefer better names here, so the next person can decipher what we did :)